types/react/canary.d.ts

```diff
@@ -32,4 +32,7 @@ type VoidOrUndefinedOnly = void | { [UNDEFINED_VOID_ONLY]: never };
 
 declare module "." {
     export function unstable_useCacheRefresh(): () => void;
+
+    export interface CacheSignal {}
```

Comment:
Is this fully renderer agnostic or do we expect that each renderer implements some minimal interface? `react-dom` could still expose the full `AbortSignal` instead of just `CacheSignal`